tools/precompute/moments_sympy.py

```diff
@@ -0,0 +1,100 @@
+"""
+This script uses SymPy to generate the analytical equations used to transform
+raw moments into central moments in skimage/measure/_moments_analytical.py
+"""
+
+from sympy import symbols, binomial, Sum
+from sympy import IndexedBase, Idx
+from sympy.printing.pycode import pycode
+# from sympy import init_printing, pprint
+# init_printing(use_unicode=True)
+
+# Define a moments matrix, M
+M = IndexedBase('M')
+
+ndim = 3
+order = 3
```

Comment:
No, this whole file is just here for future reference into how the equations in `_moments_analytical.py` were obtained. It will not be distributed with scikit-image. The idea is that a dev would just run `python moments_sympy.py` as a standalone script and it will print equations for all orders <= `order` at the specified `ndim` to the terminal. One could copy the text output from the console and paste it into `_moments_analytical.py` to reproduce expressions there.

Reply:
Oh, ok! That's why you wrote "in the GitHub repository" (file `tools/precompute/moments_sympy.py` won't be in the scikit-image package). Thanks for clarifying. :pray: